docs/sync.js

```diff
@@ -421,6 +421,30 @@ class User {
      */
     get customData() { }
 
+    /**
+     * Calls the named function as this user.
```

Comment:
Perhaps expand this description to say it's a remote function defined in MongoDB Realm Cloud?

Reply:
Does `Calls the named server function as this user.` work for you? Using the full name just reads like ad copy in the middle of the docs 🙂

Reply:
yeah - "server" at least let's the user know it's not just a local function :-)